### PR TITLE
Homework7: Alexey Kozhan

### PR DESCRIPTION
**Pull request by AlexKozhan**
Did homework in 3 Python Files

### CODE REVIEW

File: homework1.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+while True:
+    Chislo_Igroka = input("Введи 4-значное число с неповторяющимися цифрами ")
+    """Checking input number to have 4 unique digits"""
+    if (len(Chislo_Igroka) != 4 or not Chislo_Igroka.isdigit()
+            or len(set(Chislo_Igroka)) != 4):
+        print('Вы ввели некорректное число, попробуйте еще раз')
+        continue
+
+    KOLVO_BYKI = 0
```
[the remark this answers is not in the document]
Comment (AlexKozhan) [reply homework1.py:39]: > Why is this variable in the uppercase?

there was linter error with uppercase

[the remark this answers is not in the document]
Comment (AlexKozhan) [reply homework1.py]: homework1.py:39:4: C0103: Constant name "kolvo_byki" doesn't conform to UPPER_CASE naming style (invalid-name)

[the remark this answers is not in the document]
Comment (AlexKozhan) [reply homework1.py]: > Why is this variable in the uppercase?

homework1.py:39:4: C0103: Constant name "kolvo_byki" doesn't conform to UPPER_CASE naming style (invalid-name)

File: homework1.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+# Вы выиграли!
+
+Chislo_PK_List = random.sample('0123456789', 4)
+"""Generated list of 4 random unique numbers
+and changing type list to type string"""
+CHISLO_PK = ''.join(Chislo_PK_List)
+print("Загадано")
+
+while True:
+    Chislo_Igroka = input("Введи 4-значное число с неповторяющимися цифрами ")
```
[the remark this answers is not in the document]
Comment (AlexKozhan) [reply homework1.py]: > chislo_igroka

fixed

File: homework2.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+# *********
+# Необходимо написать программу, которая
+# генерирует такую пирамиду
+# со значением N, равным 10
+
+N = 10
+"""creating stars and white spaces for piramida"""
+for i in range(N):
+    stars = 2*i+1
+    probeli = N-i-1
```
Comment (Stml89) [inline homework2.py:28]: change 'probeli' to spaces. Please start use English language

File: homework3.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+Statui_List = [6, 2, 3, 8]
+Min_Statuya = min(Statui_List)
+Max_Statuya = max(Statui_List)
+Missing_Statui = []
+"""Finding elements not in Statui_List"""
+for i in range(Min_Statuya, Max_Statuya + 1):
+    if i not in Statui_List:
+        Missing_Statui.append(i)
+MISSING_STATUI_STRING = ', '.join(map(str, Missing_Statui))
+print(f"Не хватает {len(MISSING_STATUI_STRING)} статуй")
```
Comment (Stml89) [inline homework3.py:22]: it will count the whole string, e.g.: comma and spaces

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: homework3.py (hunk 4 above)
Comment (AlexKozhan) [reply homework3.py]: fixed

> it will count the whole string, e.g.: comma and spaces

File: homework2.py (hunk 3 above)
Comment (AlexKozhan) [reply homework2.py]: > change 'probeli' to spaces. Please start use English language

fixed

**Review by Stml89 — APPROVED** [review]

**Pull request merged**